scripts/random_lifetime_extension.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+import random
+import sys
+
+def generate_input(input_path, output_path, orig_lifetime, country):
+    """This function takes a full Cyclus input xml file,
+        finds the DeployInst block for the country,
+        and extends the lifetime of LWR reactors 
+        according to a Gaussian distribution (mean 10, sd 3 [years])
+
+        Parameters
+        ----------
+        input_path: str
+            path for file to be copied and modified
+        output_path: str
+            path for the modified file
+        orig_lifetime: int
+            original lifetime for lwrs (to search for)
+        country: str
+            country (region) to make this change in
+        """
+    trig = False
+    trig2 = False
+    new_file = open(output_path, 'w')
+    default_file = open(input_path, 'r')
+
+    for line in default_file:
+        if country in line and 'government' in line:
+            trig = True
+        if trig and 'lifetime' in line:
+            trig2 = True
+        if trig2:
```

Comment:
Should there be a condition for this if statement?

Reply:
yeah if trig2 is true, it does the following : 
1. it adds the lifetimes according to a random normal distribution with mean 10 and sd 6. 
2. if </lifetimes> is in line (end of listing of lifetimes) it stops.

Reply:
yeah if trig2 is true, it does the following : 
1. it adds the lifetimes according to a random normal distribution with mean 10 and sd 6. 
2. if </lifetimes> is in line (end of listing of lifetimes) it stops.

Reply:
This is a confusing structure.

Reply:
does line 32 not have to be: 
```
if trig2 = True:
```

Reply:
I made the edits to the structure, I feel like I made it more complicated. But please let me know!
@gwenchee `trig2` is a boolean so you can just do:
```
if [boolean]:
```
for a conditional!